scripts/insert_into_appointments.sql

```diff
@@ -1 +1,36 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments(start_at, length, doctor_id, patient_id)
+  SELECT TIMESTAMP '2016-08-12 10:00', p.id, d.id
```

Comment:
This will cause an error because the previous line indicates that you're inserting into four columns but here you're selecting three values.